webui/react/src/components/Spinner.module.scss

```diff
@@ -3,6 +3,11 @@
   to { transform: rotate(360deg); }
 }
 
+@keyframes rotateCentered {
+  from { transform: translate(-50%, -50%) rotate(0deg); }
+  to { transform: translate(-50%, -50%) rotate(360deg); }
+}
+
```

Comment:
**chore:** ah yes, do you want to just use this to overwrite `@keyframes rotate`?